### ISSUE TITLE
Exporting the component

### ISSUE THREAD

**Issue by kevwang424** [learn-co-curriculum/react-props-lab#8]
In the instructions it says to use:

> export default Spaceship;

However, that is not allowing the tests to pass. I had to do:

> module.exports = Spaceship

After I changed it all the tests passed.

**Comment by Lukeghenco** [comment]
Hi @kevwang424 would you be able to verify that you have the most recent version of this lab at https://github.com/learn-co-curriculum/react-props-lab . I am unable to duplicate this issue. This is the code that we are using in our solution branch. 

```jsx
import React from 'react';

class Spaceship extends React.Component {
  render() {
    const { name, speed, hasRockets, colors } = this.props;

    return (
      <div>
        <h1>Spaceship Name: {name}</h1>
        <p>Ship Info:</p>
        <p>Speed: {speed}</p>
        <p>Rockets: {hasRockets ? "Yes" : "No"}</p>
        <ul>
          {colors.map((color, index) => <li key={index}>{color}</li>)}
        </ul>
      </div>
    )
  }
}

Spaceship.defaultProps = {
  speed: 'slow',
  hasRockets: false,
  colors: ['black', 'red']
}

export default Spaceship;
```

**Comment by kevwang424** [comment]
Hey Luke, 

I think the problem is that I am using the labs from my cohort Web-1116. 

I forked from this repo.
https://github.com/learn-co-students/react-props-lab-web-1116

But I am assuming that the new classes do not have specific repos for their own cohorts and just go off of the main repo. Anyways it is not a big deal since this is only for my cohort and no one else had/has a problem. I worked around it.
